File: gpt/project-python-server/lib/models/booking.dart
class Booking {
  final String firstName;
  final String lastName;
  final String birthdate;
  final String timeSlot;

  Booking({this.firstName, this.lastName, this.birthdate, this.timeSlot});

  Map<String, dynamic> toJson() {
    return {
      'firstName': firstName,
      'lastName': lastName,
      'birthdate': birthdate,
      'timeSlot': timeSlot,
    };
  }
}

File: gpt/project-nodejs-serverless-graphql/src/models/Booking.js
const mongoose = require('mongoose');

const BookingSchema = new mongoose.Schema({
  firstName: { type: String, required: true },
  lastName: { type: String, required: true },
  birthdate: { type: Date, required: true },
  selectedTime: { type: Date, required: true },
  orderNumber: { type: Number, required: true, unique: true },
  processed: { type: Boolean, default: false }
});

module.exports = mongoose.model('Booking', BookingSchema);